reggie/ingestion/preprocessor/washington_preprocessor.py

```diff
@@ -122,16 +142,26 @@ def execute(self):
         # add voter history
         df_voter = df_voter.join(df_hist)
 
+        # Add party_idenitfier dummy values, 
+        # since WA doesn't have party info
+        df_voter.loc[:, self.config["party_identifier"]] = "npa"
+
+        # Make sure all columns are present
+        expected_cols = (
+            self.config["ordered_columns"]
+            + self.config["ordered_generated_columns"]
+        )
+        df_voter = self.reconcile_columns(df_voter, expected_cols)
+        df_voter = df_voter[expected_cols]
+
```

Comment:
One thing you're missing is 2 of Tommi's newish pipeline checks: 1) "file_check" which needs you to define "expected_number_of_files" in the yaml, and 2) "column_check" which would pre-check the thing that you're concerned with here.

It's ok that you're forcing the file columns to conform, but I think we would want to do the column_check thing first and at least log if you're receiving different columns than expected, before forcing them to conform. You can see how those 2 pipeline checks are done in other states.